package/src/headless/index.ts

```diff
@@ -27,5 +27,6 @@ export const draw = (element: ReactNode, width: number, height: number) => {
   root.dom.render(ctx);
   surface.flush();
   const image = surface.makeImageSnapshot();
+  surface.dispose();
```

Comment:
I have not tested whether any functions of `JsiSkImage` require that its `surface` has not been disposed, but as far as I can tell they don't.

Reply:
Is it possible that you are using the CPU version of the API (instead of GPU?). That would explain why disposing the surface is not causing any issues there.

Reply:
Yes - I believe I am using the CPU version. I'm running on AWS Lambda and I don't believe those instances expose much in terms of GPU resources.